### DISCUSSION TITLE
An error occurred while submitting the form.

### DISCUSSION THREAD

**Discussion by philffm** [pocketbase/pocketbase#2431] [Q&A] [answered]
Dear amazing people of pocketbase,

I just upgraded from 0.11 to 0.15 / latest and wanted to set up OAuth 2.0 but whenever I try to change a setting - be it in the OAuth settings or even Email verification templates, it returns an error. 

Everything else seems to work.

Do you have an idea? (running on Docker)

Cheers & have a great weekend , Phil 👋

![image](https://user-images.githubusercontent.com/6079545/236621441-67a8cd7f-e394-474f-a2b5-707d90dadfc4.png)

<img width="1086" alt="Screenshot 2023-05-06 at 13 27 41" src="https://user-images.githubusercontent.com/6079545/236621586-0bcaa544-cdd1-4cf2-9d79-62e9bc0f47e4.png">

**Comment by ganigeorgiev** [comment, marked as answer]
Since you are running in a Docker, my guess is that it is probably the same as in https://github.com/pocketbase/pocketbase/issues/1902, aka. try to remove the `--encryptionEnv` from your docker-compose or image flags if you don't have a valid value for it.

If that's not the case, you can try starting the executable with `--debug` and/or check the failed request logs in the Admin UI > Logs for a more detailed error message.

If none of the above helps, please provide more details about your setup (eg. share your Docker file and/or docker-compose configuration).

**Reply by philffm** [reply]
That was it 🎉 Thank you very much @ganigeorgiev !
